--- a2ps-4.15.4/etc/Makefile.am.orig	2022-08-05 15:15:08.000000000 +0200
+++ a2ps-4.15.4/etc/Makefile.am	2023-06-11 19:43:05.997178943 +0200
@@ -55,7 +55,7 @@ DISTCLEANFILES = a2ps.cfg README
 
 # Building the correct a2ps.cfg
 a2ps.cfg: a2ps_cfg Makefile
-	sed "s!@libpath@!$(libpath)!" a2ps_cfg > a2ps.cfg
+	sed -e "s!@libpath@!$(libpath)!" -e "s!@apppath@!$(PSFONT_PATH)!" a2ps_cfg > a2ps.cfg
 
 # Building a timestamp to know the version.
 README: README.in Makefile
--- a2ps-4.13/etc/a2ps_cfg.in.etc	Tue Feb  8 03:24:03 2000
+++ a2ps-4.13/etc/a2ps_cfg.in	Thu Apr 27 20:49:04 2000
@@ -73,6 +73,7 @@
 # It may be useful to extend it so that a2ps can see some
 # TeX or X11 resources: it likes AFM files and PF[AB] files.
 #AppendLibraryPath: foo
+AppendLibraryPath: @apppath@
 
 
 #################################################################
