--- a2ps-4.13/lib/Makefile.in.flex	Wed Feb  9 09:33:01 2000
+++ a2ps-4.13/lib/Makefile.in	Thu Apr 27 19:00:20 2000
@@ -100,7 +100,7 @@
 INTLDEPS = @INTLDEPS@
 INTLLIBS = @INTLLIBS@
 INTLOBJS = @INTLOBJS@
-LEX = @LEX@
+LEX = @LEX@ -CFe
 LEX_OUTPUT_ROOT = @LEX_OUTPUT_ROOT@
 LIBOBJS = @LIBOBJS@
 LIBTOOL = @LIBTOOL@
--- a2ps-4.13/lib/fonts.l.flex	Wed Dec 15 18:27:01 1999
+++ a2ps-4.13/lib/fonts.l	Thu Apr 27 19:00:20 2000
@@ -178,7 +178,7 @@
 
 %}
 
-%option yylineno
+/* %option yylineno */
 %option prefix="afm"
 %option outfile="lex.yy.c"
 
@@ -524,7 +524,7 @@
   const char * filename;
   FILE * stream;
 
-  yylineno = 0;
+  /* yylineno = 0; */
 
   /* Find its AFM file: first resolve its font file name */
   filename = fonts_map_resolve_alias (job->fonts_map, font->key);
--- a2ps-4.13/src/Makefile.in.flex	Wed Feb  9 09:34:18 2000
+++ a2ps-4.13/src/Makefile.in	Thu Apr 27 19:00:20 2000
@@ -100,7 +100,7 @@
 INTLDEPS = @INTLDEPS@
 INTLLIBS = @INTLLIBS@
 INTLOBJS = @INTLOBJS@
-LEX = @LEX@
+LEX = @LEX@ -CFe
 LEX_OUTPUT_ROOT = @LEX_OUTPUT_ROOT@
 LIBOBJS = @LIBOBJS@
 LIBTOOL = @LIBTOOL@
@@ -154,7 +154,7 @@
 BUILT_SOURCES = parsessh.h
 
 YFLAGS = -dtv
-LEX = @LEX@
+LEX = @LEX@ -CFe
 # I don't really understand why I have to put srcdir here, but
 # it is needed for yacc and lex files (seems related to #line, but
 # I really don't understand why)...
--- a2ps-4.13/src/lexps.l.flex	Mon Feb  7 17:05:43 2000
+++ a2ps-4.13/src/lexps.l	Thu Apr 27 19:00:20 2000
@@ -35,7 +35,7 @@
 void pslex_initialize PARAMS ((void));
 %}
 
-%option yylineno
+/* %option yylineno */
 %option prefix="ps"
 %option outfile="lex.yy.c"
 
--- a2ps-4.13/Makefile.in.flex	Wed Feb  9 09:32:25 2000
+++ a2ps-4.13/Makefile.in	Thu Apr 27 19:00:20 2000
@@ -100,7 +100,7 @@
 INTLDEPS = @INTLDEPS@
 INTLLIBS = @INTLLIBS@
 INTLOBJS = @INTLOBJS@
-LEX = @LEX@
+LEX = @LEX@ -CFe
 LEX_OUTPUT_ROOT = @LEX_OUTPUT_ROOT@
 LIBOBJS = @LIBOBJS@
 LIBTOOL = @LIBTOOL@
