--- a2ps-4.14/lib/output.c~	2015-04-01 22:07:29.000000000 +0200
+++ a2ps-4.14/lib/output.c	2015-04-01 22:08:53.825344087 +0200
@@ -525,7 +525,7 @@
 		     expand_user_string (job, FIRST_FILE (job),
 					 (const uchar *) "Expand: requirement",
 					 (const uchar *) token));
-	output (dest, expansion);
+	output (dest, "%s", expansion);
 	continue;
       }
 
--- a2ps-4.14/lib/psgen.c~	2007-12-29 02:58:22.000000000 +0100
+++ a2ps-4.14/lib/psgen.c	2015-04-01 22:09:36.668677711 +0200
@@ -232,7 +232,7 @@
     default:
       *buf = '\0';
       ps_escape_char (job, cp[i], buf);
-      output (jdiv, (char *) buf);
+      output (jdiv, "%s", (char *) buf);
       break;
     }
   }
--- a2ps-4.14/lib/parseppd.y~	2007-12-29 02:58:21.000000000 +0100
+++ a2ps-4.14/lib/parseppd.y	2015-04-01 22:12:11.502012094 +0200
@@ -154,7 +154,7 @@
 void
 yyerror (const char *msg)
 {
-  error_at_line (1, 0, ppdfilename, ppdlineno, msg);
+  error_at_line (1, 0, ppdfilename, ppdlineno, "%s", msg);
 }
 
 /*
--- a2ps-4.14/src/parsessh.y~	2007-12-29 02:58:35.000000000 +0100
+++ a2ps-4.14/src/parsessh.y	2015-04-01 22:12:50.782012360 +0200
@@ -740,7 +740,7 @@
 void
 yyerror (const char *msg)
 {
-  error_at_line (1, 0, sshfilename, sshlineno, msg);
+  error_at_line (1, 0, sshfilename, sshlineno, "%s", msg);
 }
 
 /*
